test/index.spec.ts

```diff
@@ -48,16 +53,11 @@ describe("addForwarder", () => {
     });
     datadogCdk.addLambdaFunctions([nodeLambda]);
     datadogCdk.addLambdaFunctions([pythonLambda]);
-    const nodeLambdaLogGroupSubscriptionFilterName = createSubscriptionFilterName(
-      nodeLambda.functionArn,
-      "forwarder-arn",
-    );
-    const pythonLambdaLogGroupSubscriptionFilterName = createSubscriptionFilterName(
-      pythonLambda.functionArn,
-      "forwarder-arn",
-    );
-    expect(nodeLambda.logGroup.node.tryFindChild(nodeLambdaLogGroupSubscriptionFilterName)).not.toBeUndefined();
-    expect(pythonLambda.logGroup.node.tryFindChild(pythonLambdaLogGroupSubscriptionFilterName)).not.toBeUndefined();
+
+    const subscriptionFilters = findDatadogSubscriptionFilters(stack);
+    expect(nodeLambda.logGroup.node.tryFindChild(subscriptionFilters[0].id)).not.toBeUndefined();
```

Comment:
I like how you refactored the tests but how can you be sure that index 0 of the `subscriptionFilters` array will always be the subscription filter associated with `nodeLambda` and subsequently how index 1 will be the filter associated with `pythonLambda`

Reply:
Good question. The answer was "it works today on my machine" but for sanity's sake I factored it out to look at the children of the lambda's themselves.